ctapipe/io/hessiofilereader.py

```diff
@@ -69,8 +62,6 @@ def _generator(self):
             # it after each yield
             counter = 0
             eventstream = file.move_to_next_event()
-            if self.allowed_tels is not None:
-                self.allowed_tels = set(self.allowed_tels)
```

Comment:
why is this needed `self.allowed_tels = set(self.allowed_tels)`? Does the traitlets not take care that this is a set?

Reply:
agreed, this is redundant now - it assures it's a set